dex-load/src/main/scala/com/wavesplatform/dex/load/WavesDexLoadCli.scala

```diff
@@ -312,7 +349,8 @@ object WavesDexLoadCli extends ScoptImplicits {
     requestsFile: File = new File(s"requests-${System.currentTimeMillis}.txt"),
     dexRestApi: String = "",
     collectTime: FiniteDuration = 5.seconds,
-    wsResponseWaitTime: FiniteDuration = 5.seconds
+    wsResponseWaitTime: FiniteDuration = 5.seconds,
+    wsCheckType: Int = 1
```

Comment:
1. Numbers are not descriptive. We use sealed traits ([enums are bad](https://medium.com/@yuriigorbylov/scala-enumerations-hell-5bdba2c1216)) to describe different type of elements. See `WavesDexCli.SeedFormat` for example.
2. You changed the default behavior of Check to `checkLeaps`. This is not a good practice when you introduce new features, because it can break places where it is used. Could this break other tests? Even if not, I suggest to not change the default behavior and update the command in SC6 runner with specifying the appropriate `ws-check-type`.

Reply:
ok